src/Microsoft.Extensions.ProjectModel.Sources/Resolution/Dependency.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Extensions.ProjectModel.Resolution
+{
+    public class Dependency
+    {
+        private const string DefaultVersion = "1.0.0";
```

Comment:
Why would we ever need a default version? If we're consuming a csproj it should have valid package versions specified.